### ISSUE TITLE
(swift-2.0 branch) building Charts.xcodeproj fails (exit code 65) with Release configuration

### ISSUE THREAD

**Issue by creisterer-db** [danielgindi/ios-charts#218]
tl;dr changing SWIFT_OPTIMIZATION_LEVEL to "-Onone" fixes xcodebuild failure, but not sure why.

Building from `swift-2.0` branch, commit `807156773de48668f9ae06aba30ec5317bca89dd`
Using Xcode 7.3 beta

From root project directory:
`xcodebuild -scheme Charts -project Charts/Charts.xcodeproj/ -configuration Release`

The build fails with exit code 65:
```
** BUILD FAILED **


The following build commands failed:
	CompileSwift normal arm64 /Users/creisterer/Develop/ios-charts/Charts/Classes/Components/ChartLimitLine.swift
	CompileSwift normal arm64 /Users/creisterer/Develop/ios-charts/Charts/Classes/Data/LineRadarChartDataSet.swift
	CompileSwift normal arm64 /Users/creisterer/Develop/ios-charts/Charts/Classes/Data/CandleChartDataSet.swift
	CompileSwiftSources normal arm64 com.apple.xcode.tools.swift.compiler
(4 failures)
```

As it succeeds with the Debug configuration I went through the build configuration settings and isolated the `SWIFT_OPTIMIZATION_LEVEL` setting. Changing this from Fast (`-O`) to mirror debug (`-Onone`) fixes this but I'm not sure why.

I first noticed this trying to checkout and build the project with Carthage.

**Comment by danielgindi** [comment]
You should post in Apple forums - they have experienced some compiler crashes in Xcode 7, and fixed some, but it seems that there are more...
